### PR TITLE
Admin page

### PR DESCRIPTION
**Pull request by runnersaw**
Hey, there's an admin page now

### CODE REVIEW

**Comment by runnersaw** [comment]
:+1: to my own pr

**Comment by runnersaw** [comment]
@cwallac you were too slow on this one

File: app.js [hunk 1]
```diff
@@ -80,5 +83,9 @@ app.get("/api/sawyerssecretroute", function(req, res) {
 	res.send("Go back, this is a secret");
 });
 
+app.get("*", function(req, res) {
+	res.redirect("/#"+req.path);
```
Comment (cwallac) [inline app.js:87]: Why is this a thing?

File: public/javascripts/main.js [hunk 2]
```diff
@@ -2,276 +2,344 @@
...
+      return window.requestAnimationFrame ||
+             window.webkitRequestAnimationFrame ||
+             window.mozRequestAnimationFrame ||
+             window.oRequestAnimationFrame ||
+             window.msRequestAnimationFrame ||
+             function(/* function FrameRequestCallback */ callback, /* DOMElement Element */ element) {
+               window.setTimeout(callback, 1000/60);
+             };
+    })();
+    console.log("LOADED");
```
Comment (cwallac) [inline public/javascripts/main.js:110]: UNLOAD

File: public/pages/admin.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
+<div ng-controller="AdminController">
+	<script src="./../javascripts/main.js"></script>
+	<style type="text/css">
+		#admin-table {
```
Comment (cwallac) [inline public/pages/admin.html:4]: Pfft css in an HTML doc. 
http://giphy.com/gifs/mind-half-amatuer-WGgMcIryybEgU

File: routes/admin.js [hunk 4]
```diff
@@ -2,17 +2,13 @@ var path = require('path');
...
-admin.get("/", isLoggedIn, isAdmin, function(req, res) {
-	console.log(req.user);
-	res.sendStatus(200);
-});
+var admin = require("express").Router();
 
 admin.post("/edit", function(req, res) {
 	// this route takes an admin userId and another userId and a boolean to give or revoke admin priveledges
-	var adminId = req.body.adminId;
+	var adminId = req.user ? req.user._id : req.body.adminId;
```
Comment (cwallac) [inline routes/admin.js:11]: HELLLL YAAAAA

**Pull request merged**